spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'users/show.html.erb', type: :feature do
```

Comment:
- [ ] Please kindly make sure to test the last scenario: `Test that redirects to the user's post index page when you click on a "see all posts" button`